### DISCUSSION TITLE
What is the % next to filter pressure? Mine reads 0

### DISCUSSION THREAD

**Discussion by jkmhb** [rstrouse/nodejs-poolController-dashPanel#63] [General]
Eom.

**Comment by tagyoureit** [comment]
Clean and dirty filter pressure can be set under Settings->Bodies->Filters.  To show the current % clean or dirty it needs to be fed a value from a pressure sensor (through REM).  The percentage you see would be the % dirty and indicate when you should clean the filter.

![image](https://user-images.githubusercontent.com/1646840/190948693-198d2084-c9b0-4944-871d-386af58a88e3.png)


![image](https://user-images.githubusercontent.com/1646840/190948475-26576713-3c59-4353-9274-c244e187c92c.png)

**Reply by jkmhb** [reply]
Thanks!
